packages/mockotlpserver/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@elastic/mockotlpserver",
-  "version": "0.6.3",
+  "version": "0.6.4",
```

```suggestion
  "version": "0.7.0",
```

and then re-run `npm install`

---

done in https://github.com/elastic/elastic-otel-node/pull/632/commits/b7908bf6b725bd3c8f22a96b3783de30dd2114ef